com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkPrefab.cs

```diff
@@ -15,6 +22,23 @@ public class NetworkPrefab
         /// </summary>
         public GameObject Prefab;
 
+        public NetworkPrefabOverride Override;
+
+        /// <summary>
+        /// Used when prefab is selected for the source prefab to override value (i.e. direct reference, the prefab is within the same project)
+        /// </summary>
+        public GameObject SourcePrefabToOverride;
+
+        /// <summary>
+        /// Used when hash is selected for the source prefab to override value (i.e. a direct reference is not possible such as in a multi-project pattern)
+        /// </summary>
+        public uint SourceHashToOverride;
+
+        /// <summary>
+        /// The prefab to replace (override) the source prefab with
+        /// </summary>
+        public GameObject OverridingTargetPrefab;
+
         /// <summary>
         /// Whether or not this is a player prefab
```

Comment:
as I commented somewhere else before, I think we should kill this flag (`NetworkPrefab.IsPlayer`) entirely.
other refactor/cleanup work to be done might be removing `NetworkPrefab.Hash` getter and renaming `NetworkPrefab` as `NetworkPrefabConfig` and making it internal instead of public since it won't be exposed to users publicly over an API etc.
mostly minor points, I'll bring these up in our internal discussions again.

Reply:
This I would hold off on based on our previous discussions.
I think NetworkPrefab could end up becoming more than just configuration related.